content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md

```diff
@@ -0,0 +1,92 @@
+### INTRODUCTION
+Technology is fast growing in the world today. This recency in growth is enhancing the livelihoods of humans and their environment. This includes the building of many self-driven cars and some smart digital systems.
+A few problems in the understanding of its working algorithm have arisen. 
+   
+   These problems have triggered developers to ask questions like: "
+
+* What is the design algorithm enhancing deep learning?
+* Why are recent developed systems able to recognize humans and their environment?
+* How can devices move accurately without human beings controlling them?
+
+      An excessive volume of information (data sets) can be trained. This uses learning algorithms in deep learning. Deep learning systems tend to gather functional facts from larger datasets. This experience (learning) is in relation to human ways of getting smarter. 
+
+#### Do you think deep learning will make current technologies better?
+
+Deep learning is, without a doubt, at the pinnacle of the most current technological hierarchy. The ability to collect data is crucial for systems (information). The data acquired enables exact forecasting and establishes the DL. As a prediction analysis, comprehending the facial recognition algorithm. This analysis functions by collecting a group of consumer behavior data sets. The intelligence and accuracy rate of deep learning are immense, and they are being checked at all levels. Humans and their environments have been great benefactors of this recent smart technology.
+ The ability to fine-tune the budget and the considered problem are two enhancing variables in Deep Learning. Any optimizer’s accuracy is measured using these factors. 
+
+### Some Determinants That Transform Deep Learning Systems' Performance
+
+#### i. Taking into account the issue (considered problem) that the system will be tasked with resolving; 
+          Simply said, this refers to the task for which the system will be created. This problem, which the system must solve, is the most crucial consideration when modeling an algorithm.
+ ######  Basic problems
+* What kind of system should be built? 
+* How will the developed algorithm handle the immediate user’s demands?
+* How easily will re-engineering be made possible?
+
+#### ii. Evaluation difficulties of default hyperparameters for multiple optimisers.
+          Poor accuracy is obtained here as a result of low awareness from training the system data sets. The Adams optimiser is used in this deep system algorithm, which is adaptable. 
```

Comment:
Don't think we can state this fact without linking adequate research papers or articles.